core/domain/story_domain.py

```diff
@@ -1043,6 +1049,22 @@ def _convert_story_contents_v3_dict_to_v4_dict(cls, story_contents_dict):
                     node['outline']))
         return story_contents_dict
 
+    @classmethod
+    def _convert_story_contents_v4_dict_to_v5_dict(cls, story_contents_dict):
+        """Converts old Story Contents schema to the modern v5 schema.
+        v5 schema introduces the thumbnail_size_in_bytes fields for Story.
+
+        Args:
+            story_contents_dict: dict. A dict used to initialize a Story
+                Contents domain object.
+
+        Returns:
+            dict. The converted story_contents_dict.
+        """
+        for index in python_utils.RANGE(len(story_contents_dict['nodes'])):
+            story_contents_dict['nodes'][index]['thumbnail_size_in_bytes'] = 0
```

Comment:
No, you should be updating it with the correct value here. (I thought we discussed this in Srikanth's design doc.)

Reply:
Adding my thoughts here:
To update the right value in the conversion function, we will need the story_id as the argument (because the thumbnail GCS URL requires story_id to fetch the resource). So, if we pursue this approach of updating the right value in the conversion function, the following changes will be required:

Include story_id as argument for the _convert_story_contents_v4_dict_to_v5_dict() method
Add story_id as argument for update_story_contents_from_model() method
Add story_id as argument for _migrate_story_contents_to_latest_schema() method in story_fetchers

If the above changes are acceptable, we will be able to update the right value in the conversion function.
@seanlip WDYT ?

Reply:
Yup, we did something similar long time ago for explorations I think.

Reply:
Ok, thanks for the confirmation.
I will check and update.

Reply:
Found the [example](https://github.com/oppia/oppia/blame/e5a1ff215388707f4c00dcbc347b0bd7f2a5faea/core/domain/exp_domain.py#L3299 ), adding it for reference here.

Reply:
@srikanthkb is this updated in design doc?